path_config.go

```diff
@@ -103,6 +124,9 @@ func (b *jwtAuthBackend) pathConfigRead(ctx context.Context, req *logical.Reques
 		Data: map[string]interface{}{
 			"oidc_discovery_url":     config.OIDCDiscoveryURL,
 			"oidc_discovery_ca_pem":  config.OIDCDiscoveryCAPEM,
+			"oidc_client_id":         config.OIDCClientID,
+			"oidc_client_secret":     config.OIDCClientSecret,
```

Comment:
So in many places we've stopped returning the root secret on read, perhaps we should remove it here too?

Reply:
Sure.